olympus/main.c

```diff
@@ -41,18 +41,31 @@ Timer_Return updateSteerDataLink(void* dummy)
     if(updateHeraSteer() != COMMS_OK)
     {
         setSevenSeg(HERA_STEER_FAIL);
-        printf("UpdateHeraSteer Failed\r\n");
+        //printf("UpdateHeraSteer Failed\r\n");
```

Comment:
Are these print statements for debugging?. Do we still need them?

Reply:
This shouldn't be commented out, will fix.

Reply:
There's another one right below it as well.